doc/chromebooks/setting_gbb_flags.md

```diff
@@ -0,0 +1,200 @@
+
+enter tty and check if CCD (Closed Case Debugging) is opened
+```
+gsctool -a -I
+```
+
+if it is opened you can skip opening it
+
+but if it isn't run
+```
+gsctool -a -o
+```
+_Note. it will require you to press PP button (a fancy word for power button) a few times, if you screen turns off after pressing that button just press it twice in a row quicklu_
+
+after last press the device will shutdonw and you will have to enable developer mode again
+
+now running
+```
+gsctool -a -I
+```
+should show that ccd is open
+
+now plugin suzyqcable to you chromebook
+
+_Note. it will on work plugged into one of chromebooks ports and in one orientation_
+
+now check if it works
+
+```
+ls /dev/ttyU*
+```
+output
+```
+ttyUSB0 ttyUSB1 ttyUSB2
+```
+_Note. this may not appear on some chromebooks with non chromeos, you can connect the other end diffrent linux pc_
+ 
+_Note. you can connect the device to itself if it has 2 ports_
+
+you can disable hardware writeprotection on any linux/chromeos (as long as device gets detected) with
+
+```
+echo "wp false" > /dev/ttyUSB0
+echo "wp false atboot" > /dev/ttyUSB0
+```
+
+_Note. on debian devices that detect the ttyUSB0 you can use ```minicom /dev/ttyUSB0``` (```sudo apt install minicom```) to see whats going on behind the sceen and type the samec commands, if it complains about modem being missing just ```ln /dev/ttyUSB0 /dev/modem```_
+
+after doing it enable it back just to be safe
+
+```
+echo "wp true" > /dev/ttyUSB0
+echo "wp true atboot" > /dev/ttyUSB0
+```
+
+sources :
+- https://www.chromium.org/chromium-os/developer-information-for-chrome-os-devices
+- https://wiki.mrchromebox.tech/Firmware_Write_Protect#Disabling_WP_on_CR50_Devices_via_CCD
+- https://chromium.googlesource.com/chromiumos/platform/ec/+/cr50_stab/docs/case_closed_debugging_cr50.md
+- https://chromium.googlesource.com/chromiumos/docs/+/master/write_protection.md
+- https://github.com/jackrosenthal/ec-zephyr-build/blob/master/docs/case_closed_debugging_cr50.md
+- https://wiki.mrchromebox.tech/Unbricking
+- https://chromium.googlesource.com/chromiumos/platform/ec/+/cr50_stab/docs/case_closed_debugging_cr50.md#Option-2_OpenNoDevMode-and-OpenFromUSB-are-set-to-Always
+
+# Setting gbb flags
+
+gbb flags make it so if your chromebook battery runs dry, you won't lose access to your system
+
+in order to set gbb flags you need to [enable developermode](./preparing-chromebook.md) and disable hardware write protection
+
+1. start by booting chrome os and entering tty with ```[ctrl]``` ```[alt]``` ```[->]``` or ```[ctrl]``` ```[alt]``` ```[refresh]``` or ```[ctrl]``` ```[alt]``` ```[f2]```
+
+2. login with username chronos (no password required)
+
+3. move to directory where ```touch testfile``` works (```cd /tmp``` should be ok)
+
+4. check software write protection with ```flashrom --wp-status```
+it will probaly enable, you can disable it with ```flashrom --wp-disable``` and check it again with ```flashrom --wp-status```
```

Comment:
```suggestion
it will probaly be enabled, you can disable it with ```flashrom --wp-disable``` and check it again with ```flashrom --wp-status```
```